Resources/Prototypes/_NF/Roles/Jobs/Nfsd/bailiff.yml

```diff
@@ -25,6 +25,9 @@
   - Mercenary # Frontier
   - Captain # Frontier
   - Armory
+  accessGroups: # Frontier
+  - GeneralNfsdAccess # Frontier
+  - ExecutiveNfsdAccess # Frontier
```

Comment:
Did you test
``access``
``accessGroups``

Working on the same time? You dont need them to exists at the same time.

Reply:
yes i tested